r-package/tests/testthat/test-download.R

```diff
@@ -1,16 +1,40 @@
 
 library(dotenv)
 
-load_dot_env('keys.env') # esperando um arquivo com esse nome na pasta tests/testthat/
+set_billing_id("basedd-cava")
```

Comment:
@pedrocava tu vai querer deixar seu billing id aqui? Não é melhor deixar a pessoa que for testar colocar um?

Reply:
boa, vou mudar isso